### PR TITLE
Add 7 new Badges.

### LINKED ISSUE #86: Would love to see "Made with | Out Pants"
I am not good with making things look pretty, but it would be awesome to have one that said:
_________________________
[Made with | out pants]
--------------------------

### LINKED ISSUE #91: [proposal] It works / on my machine
Just a little Miscellaneous badge proposal :

" IT WORKS / ON MY MACHINE "

'cause I hear this a lot at work ;)

### PR DESCRIPTION
**Pull request by ashawe**
Works on | My Machine ( FIX #91 )
Made with | Out Pants ( FIX #86 )
Not a Bug | A Feature
CTRL C | CTRL V
FIXED | BUGS
IT WORKS | WHY?
OPEN | SOURCE

### CODE REVIEW

**Comment by ashawe** [comment]
I can't understand why I can't preview the files on GitHub.

**Comment by DarkWiiPlayer** [comment]
I'd use at least 5 of these; is there any reason for not merging this already?

**Comment by ashawe** [comment]
@DarkWiiPlayer I don't know man. This repo is inactive

**Comment by ashawe** [comment]
I noticed that a few hours ago some commits were made. I want to know why any PR's are not being accepted

File: src/images/badges/ctrl_c_ctrl_v.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/ctrl_c_ctrl_v.svg:1]: Please set the font color to white to work more consistently with other badges.

image

File: src/images/badges/fixed-bugs.svg [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/fixed-bugs.svg:1]: Please set the font color to white to work more consistently with other badges.

image

File: src/images/badges/it-works-why.svg [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/it-works-why.svg:1]: Please set the font color to white to work more consistently with other badges.

image

File: src/images/badges/made-with-out-pants.svg [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/made-with-out-pants.svg:1]: Please set the font color to white, and the background color to `#ef4041` to work more consistently with other badges.

image

File: src/images/badges/not-a-bug-a-feature.svg [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/not-a-bug-a-feature.svg:1]: Please set the font color to white, and the background color to #ef4041 to work more consistently with other badges.

In addition make sure you use the correct typography as outlined in the readme for the `A FEATURE` portion of the badge.

image

File: src/images/badges/open-source.svg [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ekfuhrmann) [inline src/images/badges/open-source.svg:1]: Please set the font color to white, and the background color to #ef4041 to work more consistently with other badges.

image

**Review by ekfuhrmann — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to submit this PR. First and foremost, apologies for the long lapse in handling the PR's.

I ask that you make the changes requested, and I'll get to adding it to the project as soon as possible.

For the record, I'm a big fan of a lot of these badges, so thank you for the contribution!

File: src/images/badges/open-source.svg [hunk 7]
```diff
@@ -4,9 +4,9 @@
 	 viewBox="0 0 160.7 35.1" style="enable-background:new 0 0 160.7 35.1;" xml:space="preserve"  width="160.6" height="35">
```
Comment (ekfuhrmann) [inline src/images/badges/open-source.svg:4]: Type looks to not be center aligned.

File: src/images/badges/fixed-bugs.svg [hunk 8]
```diff
@@ -10,28 +10,28 @@
 <rect x="69.5" y="0" class="st0" width="70.6" height="35"/>
```
Comment (ekfuhrmann) [inline src/images/badges/fixed-bugs.svg:10]: Type looks to not be center aligned.

File: src/images/badges/ctrl-c-ctrl-v.svg [hunk 9]
```diff
@@ -11,29 +11,29 @@
 <g>
```
Comment (ekfuhrmann) [inline src/images/badges/ctrl-c-ctrl-v.svg:11]: Type looks to not be center aligned, and the R for the first `CTRL` seems to be of the wrong typeface. It should be `Roboto Medium`.

**Review by ekfuhrmann — CHANGES_REQUESTED** [review]
Thank you so much for taking the time to update these. There are a few minor issues with a couple of the badges that I left comments on. Otherwise everything else is looking good. If you could make those adjustments I'll add this to the site.

Thanks again for the support!

**Review by ekfuhrmann — APPROVED** [review]

**Comment by ekfuhrmann** [comment]
Looks great, thanks for taking the time to create and update these badges!

**Pull request merged**